inbc-program/tests/unit/test_source_app_parser.py

```diff
@@ -15,11 +15,13 @@ def test_parse_add_arguments_successfully(self):
             ['source', 'application', 'add',
              '-gpgKeyPath', 'https://repositories.intel.com/gpu/intel-graphics.key',
```

Comment:
since this is always a URI rather than a filesystem path, I think it would be clearer in the future to change Path to URI (or Uri, depending on the caps/lower case convention) in the parameter names, XML, etc